### PR TITLE
Add Tooltip component for Farmer table

### PR DESCRIPTION
**Pull request by ninjaprodev**

### CODE REVIEW

File: src/components/Tooltip/Tooltip.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import React, { useState, useRef } from 'react'
+import { motion } from 'framer-motion'
```
Comment (papasnowee) [inline src/components/Tooltip/Tooltip.tsx:2]: unused import

File: src/components/Tooltip/Tooltip.tsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+import { Tooltip, TooltipArrow } from 'styled-tooltip-component'
```
Comment (papasnowee) [inline src/components/Tooltip/Tooltip.tsx:3]: unused import
do you use IDE?
image

File: src/components/Tooltip/Tooltip.tsx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  TooltipWrapContainer,
+  TooltipCustomInner,
+  TooltipCustomArrow,
+  TooltipCustomWrap,
+} from './styles'
+
+import { observer } from 'mobx-react'
+
+interface IProps {
+  activator: any
```
Comment (papasnowee) [inline src/components/Tooltip/Tooltip.tsx:14]: any --> JSX.Element

**Comment by papasnowee** [comment]
image
should make the tooltip higher

File: src/components/Tooltip/Tooltip.tsx [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+          setPosition([
+            tooltipRef.current.offsetHeight,
+            tooltipRef.current.offsetLeft,
+          ])
+        }}
+        onMouseLeave={() => {
+          setHidden(true)
+        }}
+      >
+        <>
```
Comment (papasnowee) [inline src/components/Tooltip/Tooltip.tsx:36]: is it required?

File: src/components/Tooltip/Tooltip.tsx [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+  TooltipCustomWrap,
+} from './styles'
+
+import { observer } from 'mobx-react'
+
+interface IProps {
+  activator: any
+}
+
+export const TooltipCustom: React.FC<IProps> = observer((props) => {
```
Comment (papasnowee) [inline src/components/Tooltip/Tooltip.tsx:17]: the component name must be the same as the file name - "ToolTip"

**Comment by papasnowee** [comment]
new js and jsx files should be replaced with ts and tsx

File: src/components/Tooltip/Tooltip.tsx (hunk 4 above)
Comment (ninjaprodev) [reply src/components/Tooltip/Tooltip.tsx]: yes, i is

Comment (papasnowee) [reply src/components/Tooltip/Tooltip.tsx]: > yes, it is
I deleted <></> and did not notice any changes. What are they needed for?

File: src/components/Tooltip/Tooltip.tsx (hunk 1 above)
Comment (papasnowee) [reply src/components/Tooltip/Tooltip.tsx]: done

File: src/components/Tooltip/index.jsx [hunk 6]
```diff
@@ -0,0 +1 @@
+export * from './Tooltip'
```
Comment (papasnowee) [inline src/components/Tooltip/index.jsx:1]: jsx --> ts

File: src/components/Tooltip/Tooltip.tsx (hunk 4 above)
Comment (ninjaprodev) [reply src/components/Tooltip/Tooltip.tsx]: sorry, thought you mean setHidden(true)

**Pull request merged**